<ui version="4.0" >
 <class>jewelrySingleProductDiscountClass</class>
 <widget class="QWidget" name="jewelrySingleProductDiscountClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>224</width>
    <height>129</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>jewelrySingleProductDiscount</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>原零售价：</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QLineEdit" name="lineEdit" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>折扣：</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QLineEdit" name="lineEdit_2" >
     <property name="text" >
      <string>1.0</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string>销售单价：</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QLineEdit" name="lineEdit_3" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
